src/screens/collectibles-home/promotion-carousel/promotion-carousel.tsx

```diff
@@ -22,7 +23,7 @@ export const PromotionCarousel = () => {
       <Carousel
         data={promotionCarouselData}
         windowSize={layoutWidth}
-        sliderWidth={layoutWidth}
+        sliderWidth={flooredLayoutWidth}
```

Comment:
should `windowSize` be `flooredLayoutWidth`?

Reply:
no